web/app.fluidity.money/contexts/EthereumProvider.tsx

```diff
@@ -82,6 +83,11 @@ const EthereumFacade = ({
         // https://github.com/WalletConnect/web3modal/issues/455
         window.Buffer = Buffer;
         break;
+      case "coin98":
+        connector = connectors.find(
+          (connector) => connector[0] instanceof Coin98Connector
+        )?.[0];
+        break;
```

Comment:
Use the Metamask connector instance and check that `isCoin98` is present on the provider on the `window.ethereum` object. Otherwise redirect them to get the wallet.